myuw/dao/instructor_schedule.py

```diff
@@ -74,11 +76,14 @@ def _get_sections_by_section_reference(section_references, term):
 
 def _set_section_from_url(sections, section_url, term):
     section = get_section_by_url(section_url)
-    course_campus = section.course_campus.lower()
-    # check if the campus specific time schedule is published
-    if course_campus not in term.time_schedule_published or\
-       term.time_schedule_published[course_campus] is True:
-        sections.append(get_section_by_url(section_url))
+    if not term.check_time_schedule_published:
+        sections.append(section)
+    else:
+        course_campus = section.course_campus.lower()
+        # check if the campus specific time schedule is published
+        if course_campus not in term.time_schedule_published or\
```

Comment:
Why does this have `not in` and not `in and` ?

Reply:
To take care an exception when the course campus is "pce"

Reply:
Ahh I see, thank you.